beanstalk_worker/worker_start_build.py

```diff
@@ -160,7 +160,8 @@ def main():
     while True:
         try:
             logger.debug("listening to start_build tube")
-            current_jobs_in_tube = int(dict(item.split(":") for item in bs.stats_tube('start_build').split('\n')[1:-1])['current-jobs-ready'])
+            current_jobs_in_tube = bs.stats_tube(
+                'start_build')['current-jobs-ready']
```

Comment:
Since this is dealing with `start_build` tube,  I'd recommend we test it on devcloud for build execution.